### PR TITLE
If provisioning fails then take a screen shot of the failed system.

### PR DESCRIPTION
**Pull request by p3ck**
also removed the $? checking since we have set +e and it won't be executed

### CODE REVIEW

File: config/Dockerfiles/ostree-boot-image/ostree-boot-image.sh [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-
-base_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+#base_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+base_dir="$( pwd )"
 
 function clean_up {
     set +e
     ara generate junit - > /home/output/logs/ansible_xunit.xml
     if [ -e "$base_dir/hosts" ]; then
+        virsh screenshot --file /home/output/logs/atomic-host-fedoraah.ppm atomic-host-fedoraah
```
Comment (scoheb) [inline config/Dockerfiles/ostree-boot-image/ostree-boot-image.sh]: any reason for this name: fedoraah ?

Comment (p3ck) [reply config/Dockerfiles/ostree-boot-image/ostree-boot-image.sh]: not really.  Originally I was doing this as a block: rescue: piece in ansible and that would be the name if I was using {{ item.key }}.  I can make it just atomic-host.ppm, would that be better?

Comment (scoheb) [reply config/Dockerfiles/ostree-boot-image/ostree-boot-image.sh]: not a big deal...atomic-host.ppm is fine.

**Comment by p3ck** [comment]
[test]

**Comment by p3ck** [comment]
[test] rsync file disappeared

**Comment by p3ck** [comment]
[test] wake up bot

**Comment by p3ck** [comment]
[merge]

**Pull request merged**